client/account-recovery/reset-password/alternate-access-form/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity, noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class AlternateAccessFormComponent extends Component {
+	static defaultProps = {
+		translate: identity,
+		label: '',
+		description: '',
+		onSkip: null,
+		onContinue: noop,
+	};
+
+	state = {
+		isSubmitting: false,
+		fieldValue: '',
```

Comment:
This may be controversial, but I'm a fan of `null` to document a missing value. If you need to avoid React warnings in the input fields, a `value={ fieldValue || '' }` avoids that.